apps/app/src/services/renderer/remark-plugins/plantuml.ts

```diff
@@ -1,15 +1,42 @@
 import plantuml from '@akebifiky/remark-simple-plantuml';
 import { Plugin } from 'unified';
+import { Parent, Node } from 'unist';
 import urljoin from 'url-join';
 
 type PlantUMLPluginParams = {
   plantumlUri: string,
-}
+};
 
 export const remarkPlugin: Plugin<[PlantUMLPluginParams]> = (options) => {
   const plantumlUri = options.plantumlUri;
-
   const baseUrl = urljoin(plantumlUri, '/svg');
 
-  return plantuml.bind(this)({ baseUrl });
+  return (tree: Parent) => {
+    const modifiedChildren: Node[] = tree.children.map((child: Node) => {
+      const { lang, value, position } = child;
+      // Check if the node is a 'plantuml' code block and has a non-empty value
+      if (lang === 'plantuml' && value !== '') {
```

Comment:
Why how to check `value !== ''` ?

Reply:
I updated the empty value and lang checking like `if (!lang || lang !== 'plantuml' || !value) `
It will return early when `lang` is not valid e.g. when typing plain text or `lang`is not `plantuml` or the value is empty string `''`
please advise